### ISSUE TITLE
Typo in the Documentation

### ISSUE THREAD

**Issue by mikulik86** [dotnet/iot#2364]
Hi, i think you have a typo in this article: https://github.com/dotnet/iot/blob/main/Documentation/raspi-pwm.md#enabling-hardware-pwm

In the note there is supposed to be 

> Adjust the previous line to be `sudo nano /boot/config.txt`

istead of 

> Adjust the previous line to be `sudo nano /boot/firmware/config.txt`

The typo also repeats later on.

**Comment by raffaeler** [comment]
Good catch! This changed right before June 2023 according to the official documentation:
https://www.raspberrypi.com/documentation/computers/config_txt.html


> Instead of the [BIOS](https://en.wikipedia.org/wiki/BIOS) found on a conventional PC, Raspberry Pi devices use a configuration file called config.txt. The GPU reads config.txt before the Arm CPU and Linux initialise. Raspberry Pi OS looks for this file in the boot partition, located at /boot/firmware/.
>
> Note: 
> Prior to Raspberry Pi OS Bookworm, Raspberry Pi OS stored the boot partition at /boot/.


I believe it's worth to mention both.
Are you interested in submitting a pull request for this fix?

**Comment by krwq** [comment]
[Triage] `/boot/firmware/config.txt` is correct and `/boot/config.txt` is the old path. We also have a note mentioning that in the README. Closing as done
